src/utils/claculator.ts

```diff
@@ -0,0 +1,7 @@
+// operations.ts
+export const calculateOperations = (num1: number, num2: number) => ({
+  adding: num1 + num2,
+  multiplication: num1 * num2,
+  division: num1 / num2,
```

Comment:
Consider handling division by zero explicitly in the `calculateOperations` function to avoid returning `Infinity`. This can be done by checking if `num2` is zero and returning a custom error message or value. [important]